numerrin_wrapper/numerrin_wrapper.py

```diff
@@ -0,0 +1,204 @@
+""" numerrin_wrapper module
+
+Wrapper module for Numerrin using native Numerrin-Python interface
+
+"""
+
+from simphony.cuds.abc_modeling_engine import ABCModelingEngine
+from simphony.core.data_container import DataContainer
+
+from .numerrin_pool import NumerrinPool
+from .numerrin_code import NumerrinCode
+from .numerrin_mesh import NumerrinMesh
+from .numerrin_templates import numname, liccode
+
+import numerrin
+
+
+class NumerrinWrapper(ABCModelingEngine):
+    """ Wrapper to Numerrin
+
+    """
+
+    def __init__(self):
+        super(NumerrinWrapper, self).__init__()
+        numerrin.initlocal("", "PYNUMERRIN_LICENSE", liccode)
+        self.pool = NumerrinPool()
+        self.code = NumerrinCode(self.pool.ph)
+        self._meshes = {}
+        self.CM = DataContainer()
+        self.SP = DataContainer()
+        self.BC = DataContainer()
+        self.CM_extensions = {}
+        self._first = True
+
+    def run(self):
+        """Run Numerrin based on CM, BC and SP data
+
+        Returns
+        -------
+        lastIterationStep : int
+            Last iteration step step taken.
```

Comment:
On second thought it might be that not every ModelingEngine should return return last iteration or lasttime. I think that it is better to change this to obey the base class definition.